scripts/datamodel.py

```diff
@@ -228,7 +227,7 @@ def load_datamodel_dump(
     if directory is None:
         raise ValueError("directory must be supplied.")
     model_files_by_type: dict[str, list[Path]] = defaultdict(list)
-    models_pattern = re.compile(r"^(\w+\.)?(container|view|datamodel)\.yaml$")
+    models_pattern = re.compile(r"^(.*\.)?(container|view|datamodel)\.yaml$")
```

Comment:
Personal preference: Not to big fan of regex, as it is hard to read and comprehend. Can this be written as plain Python?